lib/models/user.js

```diff
@@ -59,13 +147,30 @@ User.set('toObject', {
     delete ret.deactivator;
     delete ret.resetter;
     delete ret.pendingHashPass;
+    // delete ret.paymentProcessors;
+    // delete ret.defaultPaymentProcessor;
```

Comment:
These probably need to be uncommented else possible to leak payment information?